packages/flutter/lib/src/services/platform_views.dart

```diff
@@ -74,6 +74,159 @@ class PlatformViewsService {
   }
 }
 
+/// Properties of an Android pointer.
+///
+/// A Dart version of Android's [MotionEvent.PointerProperties](https://developer.android.com/reference/android/view/MotionEvent.PointerProperties).
+class AndroidPointerProperties {
+  /// Creates an AndroidPointerProperties.
+  const AndroidPointerProperties({@required this.id, @required this.toolType});
```

Comment:
this should assert that id and toolType are non-null.
Then you can mention that in the docs for the constructor. :-)